edit/service.vue

```diff
@@ -75,6 +107,12 @@ export default {
       total:   0,
     };
 
+    const harvesterAddOnConfig = {};
```

Comment:
I couldn't see where this is used, is it still needed? If so can the initialisation code somehow be gated on `showHarvesterAddOnConfig`?

Reply:
There is use, https://github.com/harvester/dashboard/blob/78cf15909f1e17e62c6f405bcf2f912ba9685f97/edit/service.vue#L113

Reply:
Add-on config related code has been reverted cause the changes have moved to this PR: https://github.com/rancher/dashboard/pull/4509